### PR TITLE
ESLint check fix + ESLint update + Externalized SCSS from components

### PR DESCRIPTION
**Pull request by davidzynda**
## What this PR changes/adds

1. Script in package.json update so we can run the test locally and ignore some files
2. yaml file update to block the PR when ESLint finds something
3. ESLint fix for all the warnings and errors
4. Externalized the SCSS from components

### CODE REVIEW

**Review by saudkhan116 — APPROVED** [review]

**Comment by matbmoser** [comment]
I checked and the scss of the components are abstracted, which is good, but I will test it just in case.
Do we need to merge this to pass the QG5?

**Comment by matbmoser** [comment]
image
I would also not use this approach, instead I would include all them in one document called sections.scss

File: src/assets/styles/components/passport/sections/batteryComposition.scss [hunk 1]
```diff
@@ -0,0 +1,76 @@
+.battery-composition {
```
Comment (matbmoser) [inline src/assets/styles/components/passport/sections/batteryComposition.scss:1]: Add in src/assets/styles/components/passport/sections.scss

File: src/assets/styles/components/passport/sections/contractInformation.scss [hunk 2]
```diff
@@ -0,0 +1,37 @@
+.contract-information {
```
Comment (matbmoser) [inline src/assets/styles/components/passport/sections/contractInformation.scss:1]: Add in src/assets/styles/components/passport/sections.scss

File: src/assets/styles/components/passport/sections/electrochemicalProperties.scss [hunk 3]
```diff
@@ -0,0 +1,48 @@
+.electrochemical-properties {
```
Comment (matbmoser) [inline src/assets/styles/components/passport/sections/electrochemicalProperties.scss:1]: Add in src/assets/styles/components/passport/sections.scss

File: src/assets/styles/components/passport/sections/generalInformation.scss [hunk 4]
```diff
@@ -0,0 +1,45 @@
+.general-information {
```
Comment (matbmoser) [inline src/assets/styles/components/passport/sections/generalInformation.scss:1]: Add in src/assets/styles/components/passport/sections.scss

File: src/assets/styles/components/passport/sections/stateOfBattery.scss [hunk 5]
```diff
@@ -0,0 +1,45 @@
+.stateOfBattery {
```
Comment (matbmoser) [inline src/assets/styles/components/passport/sections/stateOfBattery.scss:1]: Add in src/assets/styles/components/passport/sections.scss

File: src/assets/styles/main.scss [hunk 6]
```diff
@@ -2,9 +2,16 @@
...
 
 /** Components style import **/
+
 @import "components/general/footer";
 @import "components/general/header";
 @import "components/general/search";
-@import "components/passport/documents";
 @import "components/passport/documentField";
 @import "components/landing/dashboardTable";
+@import "components/landing/dashboardTable";
```
Comment (matbmoser) [inline src/assets/styles/main.scss:11]: import src/assets/styles/components/passport/sections.scss

**Review by matbmoser — CHANGES_REQUESTED** [review]
We are duplicating logic and creating for each components in the passport a file. When the components are in the same view and are sections in this case (no abstraction is possible) we need to include in one single scss style file

**Comment by davidzynda** [comment]
> 

I don't think it's necessary for QG5 but it's definitely about quality

**Comment by matbmoser** [comment]
We need to close this pull request to refactor develop

**Comment by matbmoser** [comment]
we reopen now

**Pull request closed without merging**